### ISSUE TITLE
Implement Simplified Training Objective

### ISSUE THREAD

**Issue by jackfriskDTU** [jackfriskDTU/DDPM_deep_learning_202#4]
- [ ] Implement the simplified training objective (`Lsimple`) to train the model to predict the added noise at each time step.
- [ ] Ensure the training procedure matches the description in Algorithm 1 from the paper.
- [ ] Perform gradient descent to minimize the `Lsimple` objective.
